packages/typespec-ts/test/modularUnit/scenarios/operations/pathParam/pathParamUrlTemplate.md

```diff
@@ -14,7 +14,7 @@ model KeyBundle {
 #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Foundations.Operation is necessary for Key Vault"
 @summary("The update key operation changes specified attributes of a stored key and can be applied to any key type and key version stored in Azure Key Vault.")
 @route("/keys/{key-name}/{key-version}")
-@patch
+@patch(#{implicitOptionality: false})
```

Comment:
I think we need `implicitOptionality: true` to maintain the old behavior, reading the release notes? https://typespec.io/docs/release-notes/release-2025-05-06/